scripts/snaptron/investigate_silent_junctions.R

```diff
@@ -149,7 +187,7 @@ total_silent <- length(unique(c(silent_mutations$tcga_id,
                                 silent_mut_a$tcga_id)))
 
 f_test <- data.frame(c(silent_trunc, total_silent), c(wt_trunc, total_wt))
-sink("fishers_exon4_truncation_enrichment_in_c375G-T_mutations.txt")
+sink(file.path(opt$output_folder, "fishers_exon4_truncation_enrichment_in_c375G-T_mutations.txt"))
```

Comment:
here too. Maybe we can use a sequence of lines in the beginning of this file that defines all output names